docs/INTEGRATIONS.rst

```diff
@@ -634,7 +634,7 @@ backing up is a folder I modify permanently during my workday, so I want
 incremental backups every 5mn from 8AM to 9PM monday to friday. Here is
 the line I will paste in my crontab:
 
-.. code:: cron
+.. code:: sh
```

Comment:
If cron is not a supported style, remove it instead? Because it's not a shell command but part of a config file